.github/workflows/docker.yml

```diff
@@ -86,18 +79,40 @@ jobs:
           fi
 
           # list all tags possible
-          TAGS=""
-          TAGS="${TAGS}ghcr.io/${REPO_OWNER}/${REPO_NAME}:${{ matrix.config.tag }},"
-          TAGS="${TAGS}ghcr.io/${REPO_OWNER}/${REPO_NAME}:${{ matrix.config.tag }}-${TAG_POSTFIX}"
-
-          echo "output_tags=$TAGS" >> $GITHUB_OUTPUT
-          echo "output_tags=$TAGS"  # print out for debugging
+          if [[ "${{ matrix.config.tag }}" == "cpu" ]]; then
```

Comment:
Just wondering, can we do this a bit more simple by using `sed` to remove `-cpu` from tag name? so we don't have to specify `FULLTAGS`, `LIGHTTAGS`, `SERVERTAGS`

Another approach (with less code) could be:

```
BASE_IMAGE="ghcr.io/${REPO_OWNER}/${REPO_NAME}"
FULLTAGS="${BASE_IMAGE}:full,${BASE_IMAGE}:full-cpu"
SERVERTAGS="${BASE_IMAGE}:server,${BASE_IMAGE}:server-cpu"
```

Reply:
I could try something with sed, like:

`echo '${{ matrix.config.tag }}' | sed 's/-cpu//g'`

It would remove the if statement, but in my opinion the if statement is more "readable" or expressive at first glance.

I can simplify this, still I would need at least 3 vars to make this work and keep the project tags has they are.
The PREFIX, POSTFIX and one extra TYPE for the "matrix tag"

```
if [[ "${{ matrix.config.tag }}" == "cpu" ]]; then
    TYPE=""
else
    TYPE="-${{ matrix.config.tag }}"
fi
PREFIX="ghcr.io/${REPO_OWNER}/${REPO_NAME}:"
POSTFIX="-${TAG_POSTFIX}"
```

And

```
echo "prefix=$PREFIX" >> $GITHUB_OUTPUT
echo "type=$TYPE" >> $GITHUB_OUTPUT
echo "postfix=$POSTFIX" >> $GITHUB_OUTPUT
```
with in each build-push-action for the there it would be

`tags: ${{ steps.tag.outputs.prefix }}full${{ steps.tag.outputs.type }},${{ steps.tag.outputs.output_tags }}full${{ steps.tag.outputs.type }}${{ steps.tag.outputs.postfix }}`

Or
```
FULLTAGS="${PREFIX}full${TYPE},${PREFIX}full${TYPE}${POSTFIX}"
LIGHTTAGS="${PREFIX}light${TYPE},${PREFIX}light${TYPEl${POSTFIX}"
SERVERTAGS="${PREFIX}server${TYPE},${PREFIX}server${TYPE}${POSTFIX}"
echo "full_output_tags=$FULLTAGS" >> $GITHUB_OUTPUT
echo "light_output_tags=$LIGHTTAGS" >> $GITHUB_OUTPUT
echo "server_output_tags=$SERVERTAGS" >> $GITHUB_OUTPUT
```

What do you think?

Reply:
Pushed new code, normalized all docker images, hopefully optimized docker layers.